evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -88,6 +88,7 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
                                 <div
                                     class="slider-item card-body"
                                     data-layer="2"
+                                    id="textanswer-{{ answer.id }}"
```

Comment:
What about this question from my last review?

> Secondly, the question selection in the quick view also still shows the first unreviewed question, not the one from the fragment. Is it just broken on my computer?

I still don't think it works as intended from the issue:

> [...] the review page should be shown in the view used before **and when in quick view, the text answer just edited should be shown.**

Although the id is being set here, we are not doing anything with it in the frontend. Note that it seems the answers are shown in a somewhat random order as they are returned from the database, so the feature could work sometimes on accident. I outlined what I think needs to happen in my previous review :)